modules/serverless-tech-preview-features.adoc

```diff
@@ -120,3 +120,5 @@ ifdef::openshift-dedicated[]
 
 |====
 endif::[]
+
+// ROSA table FIXME
```

Comment:
Probably need some follow up here or will there be a new issue opened for it? I don't think I'm making any changes to the table for 1.23.0?